### ISSUE TITLE
Can't authenticate due to an additional line in server's response

### ISSUE THREAD

**Issue by januzi** [PHPMailer/PHPMailer#701]
Hi

I've read troubleshooting manual, but I can't find anything that would match my problem. Anyway, here's communication log:
CLIENT -> SERVER: AUTH LOGIN
SERVER -> CLIENT: 250 *host* Hello *host* [*ip*], pleased to meet you
SMTP ERROR: AUTH command failed
SMTP Error: Could not authenticate.
CLIENT -> SERVER: QUIT
SERVER -> CLIENT: 334 *code*

As you can see server is retarded and it is sending additional line with 250. Is there a way to bypass it (e.g. resend the command and get another line) or do I have to send some strong words to the server owner?

**Comment by Synchro** [comment]
This looks more like it's out of sync than sending a wrong response - it would be good to see what happens before this part.

**Comment by januzi** [comment]
Hello again. You were right about that:
CLIENT -> SERVER: STARTTLS
SERVER -> CLIENT: 220 Begin TLS negotiation
CLIENT -> SERVER: EHLO *name*
SMTP -> get_lines(): timed-out (10 sec)
SMTP ERROR: EHLO command failed:
CLIENT -> SERVER: HELO *name*
SERVER -> CLIENT: 250 *all possible commands*

I've found that on the server's control panel there is an option allowing users to pick php version. So, I changed it from the 5.6 to the 5.5 (just to check if that would make a change) and voila: no more timeouts. 

I've also found php.ini used by the website and it looks like it is suppressing invalid cert hostname.
